akka-docs-dev/rst/scala/code/docs/stream/FlowDocSpec.scala

```diff
@@ -138,4 +138,69 @@ class FlowDocSpec extends AkkaSpec {
 
     //#flow-connecting
   }
+
+  "various ways of transforming materialized values" in {
+    import scala.concurrent.duration._
+
+    val throttler = Flow(Source(1.second, 1.second, "test")) { implicit b =>
+      tickSource =>
+        import FlowGraph.Implicits._
+        val zip = b.add(Zip[String, Int](OperationAttributes.inputBuffer(1, 1)))
```

Comment:
Yes, I mention it because these tests are crucial to verifying the API that we provide, and if ZipWith does not work “easily” here then we know what to fix ;-)